simulation/traffic_simulator/src/math/catmull_rom_spline.cpp

```diff
@@ -315,15 +315,15 @@ boost::optional<double> CatmullRomSpline::getCollisionPointIn2D(
 }
 
 boost::optional<double> CatmullRomSpline::getSValue(
-  geometry_msgs::msg::Point position, double threshold_distance, unsigned int initial_num_points,
-  unsigned int max_iteration, double tolerance)
+  geometry_msgs::msg::Point position, unsigned int initial_num_points, unsigned int max_iteration,
+  double tolerance)
```

Comment:
`unsigned int` => `std::size_t`

https://stackoverflow.com/questions/131803/unsigned-int-vs-size-t

Reply:
fixed in https://github.com/tier4/scenario_simulator_v2/pull/603/commits/71e2f34e3d6f0fa135e9708cba12d90aa0421824